src/status_im2/contexts/share/view.cljs

```diff
@@ -0,0 +1,168 @@
+(ns status-im2.contexts.share.view
+  (:require [utils.i18n :as i18n]
+            [quo2.core :as quo]
+            [react-native.core :as rn]
+            [react-native.safe-area :as safe-area]
+            [status-im2.contexts.share.style :as style]
+            [utils.re-frame :as rf]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.qr-code.qr :as qr]
+            [clojure.string :as string]
+            ;;TODO(siddarthkay) : move the components below over to status-im2 ns
+            ;; issue -> https://github.com/status-im/status-mobile/issues/15549
+            [status-im.ui.components.list-selection :as list-selection]
+            [status-im.ui.components.react :as react]))
+
+(def ^:const profile-tab-id 0)
+(def ^:const wallet-tab-id 1)
+
+(defn copy-text-and-show-toast
```

Comment:
this should be in `reg-fx` in events namespace

Reply:
resolved here : https://github.com/status-im/status-mobile/pull/15528/commits/fef2121e678eaf935e5bc7ff3dab07acd094d099